projects/packages/connection/src/sso/class-user-admin.php

```diff
@@ -261,19 +265,27 @@ public function invite_user_to_wpcom() {
 			);
 
 			if ( 200 !== wp_remote_retrieve_response_code( $response ) ) {
-				$error        = 'invalid-invite-api-error';
+				$error_code   = 'invalid-invite-api-error';
 				$query_params = array(
 					'jetpack-sso-invite-user'  => 'failed',
-					'jetpack-sso-invite-error' => $error,
+					'jetpack-sso-invite-error' => $error_code,
 					'_wpnonce'                 => $nonce,
 				);
 
+				$tracking_event_data = array(
+					'success'    => 'false',
+					'error_code' => $error_code,
+				);
+
+				$body = json_decode( $response['body'] );
+				if ( $body && $body->message ) {
```

Comment:
Maybe this could be switched to the following to account for empty strings?

```suggestion
				if ( ! empty( $body ) && ! empty( $body->message ) ) {
```